projects/aca-folder-rules/src/lib/mock/node.mock.ts

```diff
@@ -0,0 +1,19 @@
+export const dummyNodeInfo = {
+  aspectNames: ['rule:rules', 'cm:titled', 'cm:auditable'],
+  createdAt: '2022-08-16T07:58:21.416+0000',
+  isFolder: true,
+  isFile: false,
+  createdByUser: {
+    id: 'hruser',
```

Comment:
Perhaps let's not provide real data here for a mock. Even if there's no password, let's keep it generic enough.